app/index.html

```diff
@@ -109,39 +109,39 @@
 
     <section class="section__travel-benefits container">
```

Comment:
@fernandorauxa can you move the container to wrap around everything inside the section (not on the section tag itself) that way, the background color applied does not stop at 1440px, which is the container max width.

Reply:
Done!